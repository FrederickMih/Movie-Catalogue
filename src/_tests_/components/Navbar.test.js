import { BrowserRouter } from 'react-router-dom';
import { create } from 'react-test-renderer';
import Navbar from '../../components/presentation/NavBar';

test('It should renders correctly', () => {
  const Placeholder = () => (
    <BrowserRouter>
      <Navbar />
    </BrowserRouter>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
