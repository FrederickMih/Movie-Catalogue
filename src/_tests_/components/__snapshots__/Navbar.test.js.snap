// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should renders correctly 1`] = `
<nav
  className="navbar"
>
  <h1
    className="nav-title"
  >
    Movies Catalogue
  </h1>
</nav>
`;
